### ISSUE TITLE
Call for reviewers group and new co-reviewing workflow

### ISSUE THREAD

**Issue by jywarren** [publiclab/image-sequencer#656]
Fixes #173 (😆 🎉)

One year ago, I posted #173 - Building out a contributor community for Image Sequencer, and talked a lot with @tech4GT about how to do this. As you can now see, with your help, we now have a lively community around this library! 

## Community growth

Check out some of these pages to see how many people have helped build Image Sequencer, since @ccpandhare and I started the project ~3 years ago:

* Community Toolbox: https://code.publiclab.org/#r=image-sequencer
* GitHub Pulse: https://github.com/publiclab/image-sequencer/pulse/monthly
* Contributors graphs: https://github.com/publiclab/image-sequencer/graphs/contributors

43 people have now committed code to the project, and even more have participated! 😄 🎉 

## What's next: co-review workflow

To build on this community strength 🤝, we'd like to ask for people to sign up as reviewers, to help get pull requests ready for merging. In the new workflow, we'd like each PR to have:

1. ✅ 2 reviews by people from the @publiclab/is-reviewers group
2. ✅ a `ready-to-merge` label, once ready for final review by someone with merge permissions (currently @tech4GT and I, although we are interested in expanding this group soon)
3. ✅ tests if possible (not mandatory but preferred)

This doesn't mean that we should stop helping each other solve problems even if we're not in the reviewers group -- i've seen a lot of great welcoming and cooperation! But at least we'll be able to get things ready and merged faster. 👍 🙌 

## Reviewers team

So, this new @publiclab/is-reviewers team -- we'd like to ask folks to join it if you're interested! Many of you already play this role -- THANK YOU! 🎉  We'd like folks to consider, as a prerequisite to joining:

1. ✅ having 2 PRs merged
2. ✅ having helped someone else troubleshoot a PR or issue
3. ✅ having made a `first-timers-only` issue and [welcomed in a new person to make their own first contribution](https://publiclab.org/notes/warren/10-31-2016/create-a-welcoming-first-timers-only-issue-to-invite-new-software-contributors) (this is [how our community grows](https://publiclab.org/software-outreach)!)

If you've completed these, please ask here and we'll be sure to add you the new reviewers group. You'll get notifications when people request a review from you, or mention @publiclab/is-reviewers. 

## Managing too many emails!

Just a note - your inbox can become quite a mess as the community grows! Consider turning off notifications for /every event/ and just listening to if you're mentioned! 📬 

![image](https://user-images.githubusercontent.com/24359/51051079-bbba2500-15a0-11e9-8339-b7b058371d26.png)

THANK YOU for making this corner of the Public Lab coding community a warm, welcoming, and wonderful place!!! ⚡️ 🙌 👍 🎉 🎉 🎉 

@aashna27 @HarshKhandeparkar @Divy123 @sashadev-sky @MargaretAN9 @ROODAY @VibhorCodecianGupta @Mridul97 @KusioDev @okonek @JonathanXu1 @Rishabh570 @bhavayAnand9 @oorjitchowdhary @roshniRam @publiclab/reviewers @publiclab/image-sequencer-guides @Ankit-Singla @rexagod

**Comment by Divy123** [comment]
@jywarren I have fulfilled all the conditions as mentioned to join this wonderful team .
Please consider me adding to the list of reviewers.

**Comment by VibhorCodecianGupta** [comment]
@jywarren Checked the prerequisite checkboxes here as well. Having a lot of fun with Image Sequencer, would absolutely love to be a part of the co-reviewing group!

**Comment by aashna27** [comment]
yes me too 🙌 Happy to have this opportunity 😄 😃  Looking forward to being involved more !! Thanks @jywarren @tech4GT . 🎉

**Comment by Mridul97** [comment]
@jywarren I would love to be in this team, I have checked all the conditions. Thanks! :smile:

**Comment by jywarren** [comment]
Thank you all! I've added you four. 🎉 Awesome!

**Comment by jywarren** [comment]
I'd like to also note that reviewers, like any community member, have a responsibility to support and uphold our code of conduct! This is part of what makes our community a respectful and fulfilling place. Please give it a careful read! http://publiclab.org/conduct 🎉

**Comment by HarshKhandeparkar** [comment]
I would have liked to join this too but I will be super busy for the next 2 years and won't be able to do anything. Can I still join and be redundant for 2 years?

**Comment by jywarren** [comment]
I think that's fine, @HarshKhandeparkar! I'll add you too --have you also completed the prerequisites? Thank you!

**Comment by tech4GT** [comment]
It's really nice to see our community grow here, our efforts definitely paid off! 🎉❤️

**Comment by HarshKhandeparkar** [comment]
> I would have liked to join this too but I will be super busy for the next 2 years and won't be able to do anything. Can I still join and be redundant for 2 years?

Yes I have completed the prerequisites. Thanks!

**Comment by HarshKhandeparkar** [comment]
@jywarren please add me.

**Comment by jywarren** [comment]
Adding @HarshKhandeparkar. I believe @rexagod has completed these although for other Public Lab projects in some cases, but they are similarly structured JS libs. So I think it translates pretty well! Adding you both. Thanks!

If folks can explicitly click "Approve" on PRs that would be great. 

If you need reviews, please use the `needs-review` label - that way others can scan through in the PR list. If you have 2 approved reviews, please use the `ready-to-merge` label. This'll help us be more efficient!

**Comment by milaaraujo** [comment]
I can be a reviewer, @jywarren! :)

**Comment by jywarren** [comment]
Thanks, @milaaraujo ! I'll add you to the matching `plots2-reviewers` group. We'll probably do a more formal call there soon.

**Comment by Harshithpabbati** [comment]
Can I be the part of the reviewers team.
https://github.com/publiclab/image-sequencer/pull/782
https://github.com/publiclab/community-toolbox/pull/140
These are my merged pr's

**Comment by gauravano** [comment]
You need to have one more merged PR, review one PR to completion and open an FTO in this repo.

**Comment by Harshithpabbati** [comment]
Hello
I am done with 2 merges 
Created few issues
Helped one to solve the issue
Now can I be the part of reviewers team:)

**Comment by Harshithpabbati** [comment]
@gauravano can you check it out now!!

**Comment by gauravano** [comment]
Please paste the links. Thanks!

**Comment by Harshithpabbati** [comment]
@gauravano 
My pr's
https://github.com/publiclab/image-sequencer/pull/776
https://github.com/publiclab/image-sequencer/pull/782
Issues which I made:
https://github.com/publiclab/image-sequencer/issues/801
https://github.com/publiclab/image-sequencer/issues/800
https://github.com/publiclab/image-sequencer/issues/799
https://github.com/publiclab/image-sequencer/issues/798
https://github.com/publiclab/image-sequencer/pull/795

**Comment by HarshKhandeparkar** [comment]
Harshith, are you well acquainted with the codebase? Will you be able to help other contributors if needed?

**Comment by Harshithpabbati** [comment]
yeah sure i will help other contributors.

**Comment by gauravano** [comment]
Added you to the @publiclab/is-reviewers @Harshithpabbati :tada: Thank you!

/cc @jywarren @tech4GT

**Comment by subhahu123** [comment]
@jywarren  @gauravano  can i be a  reviewer ? ( A member of publiclab/is-reviewers)

**Comment by HarshKhandeparkar** [comment]
@subhahu123 I have deleted your comments in the other pr.

**Comment by gauravano** [comment]
Hi @subahu123, I can't see your issues and PRs from https://github.com/publiclab/image-sequencer/issues/. Could you please list them below? Thanks!

**Comment by Harshithpabbati** [comment]
@jywarren I want to join into the maintainers team. How can I do that ??? Are there any prerequisites to complete???

**Comment by blurry-x-face** [comment]
Hi @jywarren !
 Can I also join the reviewer team?

**Comment by jywarren** [comment]
Hi! Thank you, can you link to your PRs and issues as above? Thanks, and awesome!!

**Comment by blurry-x-face** [comment]
@jywarren 
My PRs
#1432 
#1459 
#1482 
#1513 
#1463 
#1309 

Issues which I opened
#1488 
#1452 
#1430 
#1460 
#1464 
#1431 

Thanks!

**Comment by jywarren** [comment]
Fantastic, thank you! Adding you now.

**Comment by ataata107** [comment]
Hi @jywarren I would like to be part of this team
Kindly find my issues and PR's as follows
Issues
#1370 
#1383 
#1402 
#1411 
#1429 
#1457 
#1498

PR's
#1379 
#1386 
#1395 
#1403 
#1412 
#1428 
#1442 
#1450 
#1455 
#1456 
#1458 
#1489 
#1491 
#1533

**Comment by jywarren** [comment]
Great, adding you! Thank you!
